hip/PKGBUILD

```diff
@@ -2,7 +2,7 @@
 # Maintainer: Jakub Okoński <jakub@okonski.org>
 pkgname=hip
 pkgver=3.3.0
-pkgrel=1
+pkgrel=2
```

Comment:
Can you update `hip-hcc`? We plan to remove `hip` soon, since we want to follow the upstream packaging format of `hip-hcc` and `hip-nvcc` instead of a single `hip`.